frontend/packages/client/src/components/Community/CommunityEditorProfile.js

```diff
@@ -176,9 +205,69 @@ function CommunityEditorProfile({
                 }}
               >
                 <Upload className="has-text-white" />
-                <input {...getInputProps()} />
+                <input {...getLogoInputProps()} />
+              </div>
+            )}
+            {isUpdating && (
+              <div
+                className="is-flex is-flex-direction-column is-align-items-center is-justify-content-center"
+                style={{
+                  position: 'absolute',
+                }}
+              >
+                <p className="is-size-7">Uploading...</p>
+              </div>
+            )}
+          </div>
+        </div>
+      </div>
+      <div className="columns">
+        <div className="column is-12">
+          <div
+            className="is-flex is-flex-direction-column is-align-items-center is-justify-content-center cursor-pointer rounded-lg border-dashed-dark"
```

Comment:
to keep it the same as the avatar, when the image is added I think we should remove the border, maybe something like this?
className={`is-flex is-flex-direction-column is-align-items-center is-justify-content-center cursor-pointer rounded-lg ${!bannerImage ? 'border-dashed-dark': ''}`}

<img width="348" alt="Screen Shot 2022-07-21 at 11 12 34" src="https://user-images.githubusercontent.com/7526740/180235224-27acbfab-9680-4c1e-ab5b-0e1032ad0d38.png">